app/views/greetings/hello.html.erb

```diff
@@ -0,0 +1,17 @@
+<head>
+  <link href="https://fonts.googleapis.com/css?family=Cabin" rel="stylesheet">
+</head>
+
+
+<div id="parent">
+
+  <%= image_tag("/assets/wye-logo.png") %>
+
+  <h1 class="welcome-message"><%= @message %></h1>
+
+  <div id="buttons">
```

Comment:
Even though these buttons are displayed in the attached mock, they don't seem to be mentioned in the story's A.C.

I generally lean towards adding elements such as buttons as part of a story which actually makes use of them. This way, we don't release functionality which has ui elements that don't have any behavior.

Having said that, sometimes it makes sense to take this approach. If you're building a big feature in little steps, this could be considered a checkpoint in the "yet in-progress" building phase.

Reply:
Makes sense! However since the `Start` button will redirect the user to the retros page in the next story/PR, maybe 
 we can leave it in this page and delete the `Continue` button which has no behavior throughout all the stories I've done/started so far. What do you think?

Reply:
I ended up removing the continue button because of some other comments left in https://github.com/anita040593/RetroApp/pull/2 and left the Start button because of what i said in the previous comment.

I have addressed the other comments as well so feel free to review this PR again whenever you can :)